docs/developer/order/order-fulfillment.mdx

```diff
@@ -593,6 +593,215 @@ Some settings in Saleor can affect how fulfillments are created and processed. T
 - Changes to these settings can affect the entire fulfillment workflow
 :::
 
+
+## Post-Fulfillment Actions
+
+After an order has been fulfilled, there are several actions that can be taken 
+to handle various post-fulfillment scenarios. These actions help manage returns, 
+replacements, and refunds while maintaining proper inventory and order status tracking.
+
+### Return Products
+
+The [`orderFulfillmentReturnProducts`](/api-reference/orders/mutations/order-fulfillment-return-products) 
+mutation provides a way to handle various return scenarios, including items returns, replacements, and refunds.
+
+**Available Actions:**
+
+1. **Return**
+   - Case: Customer sends back items
+   - Provide fulfilled lines ID and quantity of items to be returned
+
+2. **Replace**
+   - Case: Items are damaged or incorrect, and customer needs replacements
+   - Process: 
+     - Set `replace: true` for the fulfillment lines that need replacement
+     - Saleor automatically creates a draft order for the replacement items
+     - The draft order includes the same customer and shipping/billing details
+     - Staff needs to:
+       1. Assign a shipping method to finalize draft order
+       2. Mark the order as paid
+       3. Fulfill the replacement order
```

Comment:
Suggestion: Maybe we should mark is as a result of the action? What do you think?